flytekit/__init__.py

```diff
@@ -2,4 +2,4 @@
 
 import flytekit.plugins
 
-__version__ = '0.9.1'
+__version__ = '0.9.3'
```

Comment:
i made this 0.9.3 cuz I thought katrina's change for single-task would go in first but I think this will go in first actually.